packages/cli/src/commands/init/init.ts

```diff
@@ -177,7 +188,8 @@ async function createProject(
   version: string,
   options: Options,
 ) {
-  const templateName = options.template || `react-native@${version}`;
+  const templateName =
+    options.template || `react-native@${options.devVersion || version}`;
```

Comment:
Insn't this a limitation? if we force `react-native@{options.devVersion}`, this will not work when pointed at git repository? (useful for unpublished forks and PRs)

Reply:
We don't seem to be doing such prepending in `changeReactNativeVersionInTemplate` tho.